### ISSUE TITLE
Still active

### ISSUE THREAD

**Issue by rollsroyc3** [jwcao/opentok-titanium#7]
Im guessing this project is dead but just wanted to confirm...

**Comment by jwcao** [comment]
Correct. We've switched our mobile development to Xamarin (which I highly recommend), so I will no longer be updating this repository.

**Comment by rollsroyc3** [comment]
OK thanks, yeah this is my last Appcelerator project I have to maintain, forked if anyone needs it still.

**Comment by stemckillop** [comment]
which version of appcelerator are you using with this...im running into text relocation errors.
Thanks :D

**Comment by rollsroyc3** [comment]
I compile it against 3.5.1 but its being used in a 5.0.1 app

**Comment by rollsroyc3** [comment]
Hey @jwcao, just wanted to update you I am actively maintaining a branch with a few fixes in case you want to forward people onto it. 


I also wanted to ask if you had an opinion on the Android upgrade I am currently working on. With Opentoks Android 2.10 SDK they have a new architecture, arm64-v8a. After setting everything up and building I am missing libcom.tokbox.ti.opentok.so in the new folder arm64-v8a. This seems to be the issue of the module crashing Android at startup. Any ideas?

Here is a screenshot of the libs folders after compiling the module
https://www.dropbox.com/s/4is7ypb5h6d9525/Screenshot%202017-01-23%2015.26.00.png?dl=0
